src/InstallationValidator.Core/Assets/Captions.cs

```diff
@@ -20,21 +20,21 @@ public static string ValidationDescription
          get
          {
             var sb = new StringBuilder();
-            sb.AppendLine($"A validation of the <b>{OSPSuite.Core.Domain.Constants.SUITE_NAME}</b> installation is performed to ensure that the software works fully as intended when installed in the computing environment.");
-            sb.AppendLine();
-            sb.AppendLine("The key functionalities are tested through comparison of reference simulation outputs with locally calculated outputs for a set of predefined test simulations.");
-            sb.AppendLine();
-            sb.AppendLine("The validation report will include:");
-            sb.AppendLine("  - overall validation result (<b>Valid/ Invalid</b>)");
-            sb.AppendLine("  - validation result for every test simulation (<b>Valid/ Invalid</b>)");
-            sb.AppendLine("  - summary of the deviations (if any) for each <b>Invalid</b> simulation (numerical values and comparison plots)");
-            sb.AppendLine("  - information about installed software versiosns and local computing environment");
+            Html.AppendLine($"A validation of the <b>{OSPSuite.Core.Domain.Constants.SUITE_NAME}</b> installation is performed to ensure that the software works fully as intended when installed in the computing environment.", sb);
```

Comment:
if we want to use that more often, we will need to make creation of HTML string easier, probably as extgension of StringBuilder. Right now, a bit ugly